lib/facter/winlogbeat_version.rb

```diff
@@ -1,10 +1,13 @@
 require 'facter'
 Facter.add('winlogbeat_version') do
-  confine :kernel => %w[Windows] # rubocop:disable Style/HashSyntax
+  confine :kernel => 'Windows' # rubocop:disable Style/HashSyntax
   if File.exist?('c:\Program Files\Winlogbeat\winlogbeat.exe')
-    winlogbeat_version = Facter::Util::Resolution.exec('"c:\Program Files\Winlogbeat\winlogbeat.exe" --version')
+    winlogbeat_version = Facter::Util::Resolution.exec('"c:\Program Files\Winlogbeat\winlogbeat.exe" version')
```

Comment:
From this change, it seems like certain versions of winlogbeat use `--version` and some use `version`. If this is the case, could you add a short comment that reads something like versions before X used `--version`. That would let others know why this is here.